server/src/international_application_processor/onboarding_messages.go

```diff
@@ -0,0 +1,327 @@
+		<p>
+			I wanted to reach out about your inquiry of getting involved with
+			DxE’s international network.
+			There is currently a DxE chapter near you, so I’ve included their
+			information and contact below so you can reach out, get involved,
+			and start taking action with them!
+		</p>
+	`)
+
+	body.WriteString("<p>")
+	if b.chapter.FbURL != "" {
+		fmt.Fprintf(&body, `<a href="%v">DxE %v Facebook page</a><br />`,
+			b.chapter.FbURL, b.chapter.Name)
+	}
+	if b.chapter.InstaURL != "" {
+		fmt.Fprintf(&body, `<a href="%v">DxE %v Instagram</a><br />`,
+			b.chapter.InstaURL, b.chapter.Name)
+	}
+	if b.chapter.TwitterURL != "" {
+		fmt.Fprintf(&body, `<a href="%v">DxE %v Twitter</a><br />`,
+			b.chapter.TwitterURL, b.chapter.Name)
+	}
+	if b.chapter.Email != "" {
+		fmt.Fprintf(&body, `<a href="mailto:%v">DxE %v Email</a><br />`,
+			b.chapter.Email, b.chapter.Name)
+	}
+	body.WriteString("</p>")
+
+	body.WriteString(
+		`<p>
+			I’ve also cc'd the organizers in your local chapter on this email,
+			so you can both be in contact.
+		</p>`)
+
+	nextEventLink := getFacebookEventLinkOrEmptyString(b.nextEvent)
+	if len(nextEventLink) > 0 {
+		fmt.Fprintf(
+			&body,
+			"<p>You can also find details of our next event here: %v</p>",
+			nextEventLink)
+	}
+
+	body.WriteString(`
+		<p>
+			In the meantime you can
+			<a href="https://righttorescue.com/">sign a letter to support the
+			right to rescue</a>.
+		</p>
+		<p>
+			Let me know if you have any questions or if you still haven't been
+			able to connect with your local chapter.
+		</p>
+		<p>Hope that you can join us!</p>
+		`)
+	fmt.Fprintf(&body, `
+		<p>
+			<strong>%v</strong><br/>
+			International Coordinator<br/>
+			Direct Action Everywhere
+		</p>`,
+		globalCoordinator.Name)
+
+	msg.BodyHTML = body.String()
+
+	return msg, nil
+}
+
+func (b *onboardingEmailMessageBuilder) caOrganizerNotNearAnyChapter() (mailer.Message, error) {
+	var msg mailer.Message
+	msg.FromName = californiaCoordinator.Name
+	msg.FromAddress = californiaCoordinator.Address
+	msg.ToName = b.fullName
+	msg.ToAddress = b.email
+
+	msg.Subject = "Getting involved with Direct Action Everywhere"
+
+	var body strings.Builder
+
+	fmt.Fprintf(&body, `<p>Hi %v,</p>`, html.EscapeString(b.firstName))
+
+	body.WriteString(`
+		<p>
+			Thank you for your interest in joining the DxE Network.
+			Right now, DxE is actively seeking new organizers and chapters in
+			California.
+			There is no active chapter in your area and I am excited to help you
+			launch one!
+		</p>`)
+	fmt.Fprintf(&body, `
+		<p>
+			To begin, please find 1-2 other people in your area who might be
+			interested in helping you organize a chapter. Once you have
+			identified them, reach out to me by email at %v to schedule a call.
+			Don’t hesitate to email me with any questions in the meantime.
+			I’m looking forward to hearing back from you,
+		</p>`, californiaCoordinator.Address)
+
+	body.WriteString(almiraTannerSignatureHtml)
```

Comment:
nit: this code feels like a little bit of a footgun if we update the `californiaCoordinator` to be someone else, but forget that it includes `almiraTannerSignatureHtml`? i wonder if it's worth adding `role` to the coordinator structs & then just building the signature for everyone based on that.

Reply:
oh good idea. i felt this was a footgun but it didn't occur to me to just add a "role" field. i moved the roles for Michelle and Almira and made individual functions for each coordinator role, but i didn't make a single generic signature function because they're not all exactly the same. i don't want to make everyone standardize on whether to put their pronouns, and it's nice to allow small customizations, like Antonelle's sig says "DxE Organizer" instead of "Organizer\nDxE".

anyway hopefully we'll catch errors with manual review of the HTML allowed by mailer_integration_test.go.